airbyte-integrations/connectors/source-stripe/integration_tests/configured_catalog.json

```diff
@@ -3,43 +3,7 @@
     {
       "stream": {
         "name": "events",
-        "json_schema": {
-          "type": "object",
-          "properties": {
-            "created": {
-              "type": ["null", "integer"]
-            },
-            "data": {
-              "type": ["null", "object"],
-              "properties": {}
-            },
-            "id": {
-              "type": ["null", "string"]
-            },
-            "api_version": {
-              "type": ["null", "string"]
-            },
-            "object": {
-              "type": ["null", "string"]
-            },
-            "livemode": {
-              "type": ["null", "boolean"]
-            },
-            "pending_webhooks": {
-              "type": ["null", "integer"]
-            },
-            "request": {
-              "type": ["null", "string"]
-            },
-            "type": {
-              "type": ["null", "string"]
-            },
-            "updated": {
-              "type": ["null", "string"],
-              "format": "date-time"
-            }
-          }
-        },
+        "json_schema": {},
```

Comment:
In normalization, we expect streams in catalog to always have `json_schema.properties` otherwise we raise exceptions:
https://github.com/airbytehq/airbyte/blob/953744a83a64c3abb28cb8d514db08ad57be9313/airbyte-integrations/bases/base-normalization/normalization/transform_catalog/catalog_processor.py#L143

Why are we sending streams with empty schemas for?

Reply:
@ChristopheDuong 
In this case, we use `configured_catalog.json` for Acceptance Tests, and in order not to duplicate the scheme once again, a processing has been created that pulls up the scheme for each stream from the streams directory.
You can see it [here](https://github.com/airbytehq/airbyte/blob/953744a83a64c3abb28cb8d514db08ad57be9313/airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py#L83).

In your case, we get a directory that is located in the [`f"resources/{catalog_file}.json"`](https://github.com/airbytehq/airbyte/blob/953744a83a64c3abb28cb8d514db08ad57be9313/airbyte-integrations/bases/base-normalization/unit_tests/test_table_name_registry.py#L85) file. I am not very competent in this code, since I have not worked with it, but I suspect that these catalogs were created using the `discover` command and placed in the `resources/` directory, and `json_schema` is always present there.

Reply:
Oh i see, that's how the tests are done here, ok!

I was just worried that sources would produce streams with empty properties as it already happened in the past as shown recently in this thread:
https://airbytehq.slack.com/archives/C01MFR03D5W/p1627044823226100

Reply:
No no, this is a different case. This practice of using `catalogs` without `json_schema` for Acceptance Tests is not new, it has been used for about month for sure with different connectors.